G1ANT.Addon.Xlsx/G1ANT.Addon.Xlsx.Tests/Commands/xlsx.close/ShouldCloseFile_WhenOpenFile.G1ANT

```diff
@@ -0,0 +1,24 @@
+﻿addon xlsx version 2.205.17304.1000
+addon core version 4.99.18340.1354
+addon language version 4.99.18340.1354
+
+♥folderPath = ♥environment⟦TEMP⟧\
+♥filePath = ♥folderPath + "TempTestDebug.xlsx"
+
+procedure DeleteTestFileIfExists
+    ⊂
+        if(System.IO.File.Exists(♥filePath))
+        {
+            System.IO.File.Delete(♥filePath);
+        }
+    ⊃
+end procedure
+
+procedure CreateFileWhenOpening
+    xlsx.open path ♥filePath createifnotexist true
+    xlsx.close
+end procedure
+
+call DeleteTestFileIfExists
+call CreateFileWhenOpening
+call DeleteTestFileIfExists
```

Comment:
xlsx.close do not return anything, and we can't check opened files
so if this test don't throw exception than test pass

Reply:
Nope. You don't verify that the file was created.

Also, you could use `file.delete` instead of `DeleteTestFileIfExists`, it already does not throw an exception if the file does not exist.
The test could be rewritten to:
```g1ant
♥filePath = ♥environment⟦TEMP⟧\TempTestDebug.xlsx
file.delete ♥filePath

xlsx.open path ♥filePath createifnotexist true
xlsx.close

file.exists ♥filePath timeout 0 errorjump onExpectedException
test false errormessage ‴The file ♥filePath has no been created‴
label onExpectedException
file.delete ♥filePath
```

Reply:
done